### PR TITLE
Updated vagrant demo

### PR DESCRIPTION
**Pull request by kayrus**
/cc @robszumski

### CODE REVIEW

**Comment by robszumski** [comment]
LGTM but @crawford can you take a quick look too?

File: config.rb.sample [hunk 1]
```diff
@@ -14,18 +14,20 @@ $new_discovery_url="https://discovery.etcd.io/new?size=#{$num_instances}"
...
 #  if data['coreos'].key? 'etcd2'
 #    data['coreos']['etcd2']['discovery'] = token
 #  end
 #
 #  # Fix for YAML.load() converting reboot-strategy from 'off' to `false`
-#  if data['coreos']['update'].key? 'reboot-strategy'
-#     if data['coreos']['update']['reboot-strategy'] == false
-#          data['coreos']['update']['reboot-strategy'] = 'off'
-#       end
+#  if data['coreos'].key? 'update' and data['coreos']['update'].key? 'reboot-strategy'
```
Comment (crawford) [inline config.rb.sample]: Does this also need `data.key? 'coreos'`?

Comment (kayrus) [inline config.rb.sample]: Can not see anything against that. And also now we can completely uncomment that code to allow automatic discovery substitution by default.

**Comment by kayrus** [comment]
Any news?

**Comment by kayrus** [comment]
@robszumski @crawford  bump

**Comment by crawford** [comment]
Cloud-Config YAML (tm) is such a mess. LGTM. Please squash.

**Comment by kayrus** [comment]
Squashed

**Comment by kayrus** [comment]
Any news?

**Comment by kayrus** [comment]
@robszumski updates?

**Pull request merged**